python/python/glide/async_commands/core.py

```diff
@@ -500,6 +500,9 @@ async def rename(self, key: str, new_key: str) -> TOK:
         meaning that in practice only keys that have the same hash tag can be reliably renamed in cluster.
         See https://redis.io/commands/rename/ for more details.
 
+        Note:
+            When in cluster mode, the command may route to multiple nodes when `keys` map to different `hash slots`.
```

Comment:
why is `hash slots` in backticks?  This is not related to Python. 
We should only backtick Python-related variables and terms.

Reply:
we do that in java... ok I'll remove
Do you want me to remove that in java too?